libvast/src/system/stop_command.cpp

```diff
@@ -0,0 +1,58 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#include "vast/system/stop_command.hpp"
+
+#include "vast/detail/narrow.hpp"
+#include "vast/error.hpp"
+#include "vast/logger.hpp"
+#include "vast/scope_linked.hpp"
+#include "vast/system/connect_to_node.hpp"
+
+#include <caf/actor.hpp>
+#include <caf/scoped_actor.hpp>
+#include <caf/settings.hpp>
+
+#include <iostream>
+
+using namespace std::chrono_literals;
+
+namespace vast::system {
+
+caf::message
+stop_command(const command::invocation& invocation, caf::actor_system& sys) {
+  VAST_TRACE(invocation);
+  // Bail out early for bogus invocations.
+  if (caf::get_or<bool>(invocation.options, "system.node", false))
```

Comment:
```suggestion
  if (caf::get_or(invocation.options, "system.node", false))
```

The template parameter can be derived from the fallback value.